Jenkinsfile.aws-test

```diff
@@ -59,6 +59,9 @@ node(NODE) {
                     fi
 
                     cp ${dirpath}/aws-${AWS_REGION}-smoketested.json ${images}/aws-${AWS_REGION}-tested.json
+                    aws ec2 create-tags \
+                        --resources \${ami_intermedia} \
+                        --tags rhcos_test=smoketested
```

Comment:
This is bikeshed, though maybe this should just be `tested`? Given that right now we use smoketested to refer to the first kola run in the cloud job.

Reply:
> This is bikeshed, though maybe this should just be `tested`?

+1 to generic names until we have several test classes to distinguish between.  If we already have plans for more specific tests, than +1 to namespacing with `smoke` now ;).

Reply:
Beating the horse further...we discussed tagging in #201 and #150 

There seemed to be consensus around https://github.com/openshift/os/issues/201#issuecomment-423214423

While that is about the container tags, there was agreement that the container tags and image tags should be the same.

Reply:
OK right so...based on that comment you're saying the tag should be `alpha` right?

Reply:
I updated with that change :arrow_down: